content/md/products-super-power/product-mass-actions.md

```diff
@@ -15,18 +15,19 @@ To access the bulk actions, go on the Product grid, **select at least one produc
 
 # Available bulk actions
 Here’s the list of the bulk operations available in the PIM (Community and Enterprise Editions):
-  - **Edit attributes values**: to edit the attributes values of your selection of products. For instance, set a new release date or a new value for an attribute. This action does not add attribute to products
-  - **Add attributes values (_since 2.2_)**: to add values for the multivalued attributes on your selection of products. For instance, add a collection or an asset. This action does not add attribute to products  
-  - **Change family**: to choose/change the family of the selected products
-  - **Change status (enable/disable)**: this action allows you to edit the status of one or several products, to enable or disable the products
-  - **Add to groups**: to add your selection of products in one or more groups
-  - **Add to categories**: to add product into categories
-  - **Move between categories**: to change your product categorization
-  - **Remove products from categories**: to remove products from one or more categories at once. Selected products will be uncategorized.
-  - **Publish products (Enterprise Edition only)**: to mass publish products
-  - **Unpublish products (Enterprise Edition only - available from the Published product grid)**: to mass unpublish products
-  - **Add to an existing product model**: to add products as variant of a [product model](/articles/what-about-products-variants.html#what-is-a-product-model)
-  - **Associate to products (_since 2.1_)**: to add associations for your selection of products
+  - **Edit attributes values**: Edit the attributes values of your selection of products. For instance, set a new release date or a new value for an attribute.
+  - **Add attributes values (_since 2.2_)**: Add values for the multivalued attributes on your selection of products. For instance, add a collection or an asset.  
+  - **Change family**: Choose/change the family of the selected products.
+  - **Change status (enable/disable)**: Edit the status of one or several products, to enable or disable the products.
+  - **Add to groups**: Add your selection of products in one or more groups.
+  - **Add to categories**: Add products or product models into categories.
+  - **Move between categories**: Change your products categorization
+  - **Remove products from categories**: Remove products from one or more categories at once. Selected products will be uncategorized.
+  - **Add to an existing product model**: Add products as variant of a [product model](/articles/what-about-products-variants.html#what-is-a-product-model).
+  - **Change the parent product model (_since 2.3_)**: Change the parent [product model](/articles/what-about-products-variants.html#what-is-a-product-model) of variant products or product models.
```

Comment:
in the last sentence, do you use "product model" for sub product model as you don't want the subproduct model concept to appear in the documentation?

Reply:
That's what I guessed, it just sounds weird to me to "change the parent product model of a product model". Add a link to the parent model definition may be enough.